services/skus/service.go

```diff
@@ -136,6 +143,7 @@ type Service struct {
 	geminiClient       gemini.Client
 	geminiConf         *gemini.Conf
 	scClient           *client.API
+	stpClient          stripeClient
```

Comment:
nit: the field name seems a bit awkward

suggestion what about `stripeCl` or something similar?

Reply:
Thanks, have updated it to `stripeCl`.